app/views/questions/eip_overview/edit.html.erb

```diff
@@ -26,7 +26,14 @@
       </li>
     </ul>
 
-    <button class="button button--primary button--wide text--centered spacing-above-60" name="eip_overview_form[eip_only]" value="true" type="submit">
+    <div class="notice--warning text--small">
+      <p class="spacing-below-25 text--bold"><%=t("views.questions.eip_overview.hotline_notice.title") %></p>
+      <p><%=t("views.questions.eip_overview.hotline_notice.what") %></p>
+      <p><%=t("views.questions.eip_overview.hotline_notice.why") %></p>
+      <p class="text--underline"><%=t("views.questions.eip_overview.hotline_notice.number") %></p>
```

Comment:
I noticed that you underlined this. That would make it look like a link. You could make it into an actual link that allows a user to initiate a phone call.
```html
Call <a href="tel:+18443223639">+1 844 322 3639</a>
```

Reply:
oh that makes much more sense thanks